cmd/upgrade.go

```diff
@@ -276,7 +278,7 @@ func (d *diffCmd) run() error {
 		}
 	}
 
-	seenAnyChanges := diff.Manifests(currentSpecs, newSpecs, d.suppressedKinds, d.showSecrets, d.outputContext, d.output, d.stripTrailingCR, os.Stdout)
+	seenAnyChanges := diff.Manifests(currentSpecs, newSpecs, d.suppressedKinds, d.showSecrets, d.outputContext, d.maxChangeFraction, d.output, d.stripTrailingCR, os.Stdout)
```

Comment:
Can you make the rename detection get triggered only when `newInstall` is `false`? In L226-L228 we already call helm to detect if it's a diff for an upgrade(=there can be renames) or not(=there's no rename by the nature of it's a new release).

That way, I think O(N^2) issue won't affect new install badly.

Reply:
Sure, I'll change it